types/react-simple-tooltip/package.json

```diff
@@ -0,0 +1,20 @@
+{
+    "private": true,
+    "name": "@types/react-simple-tooltip",
+    "version": "2.6.9999",
+    "projects": [
+        "https://github.com/cedricdelpoux/react-simple-tooltip#readme"
+    ],
+    "dependencies": {
+        "@types/react": "*"
+    },
+    "devDependencies": {
+        "@types/react-simple-tooltip": "workspace:."
+    },
+    "owners": [
+        {
+            "name": "ClassDome Technologies",
+            "githubUsername": "classdome"
```

Comment:
Why isn't this you?

Reply:
I am submitting this PR in the name of my corporation, ClassDome Technologies, and, in the readme, it is explained that Corporations can also be used as owners.

Reply:
> that Corporations can also be used as owners

IMO the readme says: `DT has the concept of "Definition Owners" which are people ...`. The main purpose is to manage PRs, organisation afaik cannot do this.

Reply:
The example given tight below that does specify one as an owner, though.

Reply:
If you're the author of the underlying package, you should ship the types as [part of your library](https://www.typescriptlang.org/docs/handbook/modules/guides/choosing-compiler-options.html#im-writing-a-library) instead of putting them on DefinitelyTyped. This is generally better for users (they don't have to install a separate `@types` pacakge), better for DT (keeps the repo size down), and better for you (single-step publishing and better versioning coherence). If you have a particular reason this doesn't work for you, please explain and we can reconsider.

Reply:
Okay, I think I'm explaining myself badly here so let me clear some things up:
1. I am **not** the owner of the underlying package, nor is my organization.
2. The entity in charge of maintaining these types would be ClassDome Technologies, not me, though I am the lead dev there.
3. If having only an Org as the owner is a problem, I can totally add myself to the list as well.

I apologize if i haven't been clear in previous comments. I hope this clears things up.

Reply:
Ah, I had assumed Classdome was the one shipping react-simple-tooltip for some reason. I see now. Sorry for the confusion